pom.xml

```diff
@@ -135,27 +135,27 @@
     <log4j.over.slf4j.version>1.7.22</log4j.over.slf4j.version>
     <luaj.version>3.0.1</luaj.version>
     <maven.version>3.3.9</maven.version>
-    <metrics.client.version>0.7.0</metrics.client.version>
+    <metrics.client.version>0.10.0</metrics.client.version>
     <metrics.jvm.extra.version>0.7.0</metrics.jvm.extra.version>
     <metrics.http.extra.version>0.6.0</metrics.http.extra.version>
```

Comment:
You should probably release new version of jvm metrics and http extra since those build deps were just merged.